lib/ansible/modules/cloud/azure/azure_rm_webapp.py

```diff
@@ -745,6 +746,10 @@ def exec_module(self, **kwargs):
 
                 self.results['id'] = response['id']
 
+            if self.to_do == Actions:UpdateAppSettingss:
```

Comment:
I think there is a syntax error here.
```suggestion
            if self.to_do == Actions.UpdateAppSettings:
```

Has this been tested?

Reply:
sorry for the quick pr. I've updated after testing.